File: backend/src/main/java/com/ll/hotel/domain/review/comment/dto/ReviewCommentDto.java
package com.ll.hotel.domain.review.comment.dto;

import com.ll.hotel.domain.review.comment.entity.ReviewComment;

import java.time.LocalDateTime;

public record ReviewCommentDto(
        long reviewCommentId,
        String content,
        LocalDateTime createdAt
) {
    public ReviewCommentDto(ReviewComment reviewComment){
        this(reviewComment.getId(), reviewComment.getContent(), reviewComment.getCreatedAt());
    }
}

File: backend/src/main/java/com/ll/hotel/domain/review/comment/dto/ReviewCommentDto.kt
package com.ll.hotel.domain.review.comment.dto;

import com.ll.hotel.domain.review.comment.entity.ReviewComment;

import java.time.LocalDateTime;

data class ReviewCommentDto(
    val reviewCommentId: Long,
    val content: String,
    val createdAt: LocalDateTime
) {
    constructor(reviewComment: ReviewComment) : this(
        reviewComment.id,
        reviewComment.content,
        reviewComment.createdAt
    );
}
